src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureKeyVaultReference/CachedKeyVaultSecret.cs

```diff
@@ -17,10 +17,16 @@ internal class CachedKeyVaultSecret
         /// </summary>
         public DateTimeOffset? RefreshAt { get; set; }
 
-        public CachedKeyVaultSecret(string secretValue, DateTimeOffset? refreshAt)
+        /// <summary>
+        /// The number of times we tried to reload this secret with exponential backoff strategy.
```

Comment:
> with exponential backoff strategy

I don't think attemp tracking needs to be coupled with backoff implementation.

Reply:
removed